PowerEditor/visual.net/notepadPlus.vcxproj

```diff
@@ -594,7 +593,6 @@ copy ..\src\contextMenu.xml ..\bin64\contextMenu.xml
     <ClInclude Include="..\src\ScitillaComponent\DocTabView.h" />
     <ClInclude Include="..\src\WinControls\DocumentMap\documentMap.h" />
     <ClInclude Include="..\src\EncodingMapper.h" />
-    <ClInclude Include="..\src\WinControls\OpenSaveFileDialog\FileDialog.h" />

```

Comment:
Shouldn't we add `..\src\WinControls\OpenSaveFileDialog\CustomFileDialog.h`?

Reply:
It is already there, see https://github.com/notepad-plus-plus/notepad-plus-plus/blob/611790310c89e1bf1357b1be7a9fdfd3abbebd68/PowerEditor/visual.net/notepadPlus.vcxproj#L579